packages/riipen-ui/src/components/Popover.jsx

```diff
@@ -126,21 +143,32 @@ class Popover extends React.Component {
   componentDidUpdate(prevProps) {
     const anchorChange =
       prevProps.anchorEl === null && this.props.anchorEl !== null;
-    const openChange = !prevProps.isOpen && this.props.isOpen;
+
+    /* need to NOT check that previous prop was !isOpen
+     * in order to cause positioning of popover to be correct
+     * when it is opened by a controlled Tooltip being opened
+     */
+    const openChange = this.props.isOpen;
     const anchorPositionChange = !(
       prevProps.anchorPosition === this.props.anchorPosition
     );
     const contentPositionChange = !(
       prevProps.contentPosition === this.props.contentPosition
     );
 
+    if (anchorChange) {
+      // add event listeners
+      this.manageScrollableParentEventListeners(false);
+    }
+
     if (
       anchorChange ||
       openChange ||
       contentPositionChange ||
       anchorPositionChange
     ) {
-      this.setPositioningStyle();
+      // need to use requestAnimationFrame in order for popover to get proper position on first open
+      window.requestAnimationFrame(this.setPositioningStyle);
```

Comment:
this fixes initial positioning `Popover` bug, but only when the `Popover` is opened normally (with MouseEnter / Click), need changes on `L151` to fix bug when `Popover` is opened via 'changing tab' in an `InteractiveTooltip`